static/js/gio_calendar_quad.js

```diff
@@ -17,7 +17,7 @@ function retrieveNextFourEvents() {
       // if a community meeting, apply meeting-box to classname
       var classname = e.community_meeting ? 'col-md-3 meeting-box' : 'col-md-3';
       var communityMeetingStar = e.community_meeting ? ` <i class="glyphicon glyphicon-star" style="color: #1e8dc1;"></i>` : '';
-      var communityMeetingAgenda = e.community_meeting ? ` <a class="btn btn-tnris btn-md" href="https://tnris-org-static.s3.amazonaws.com/documents/GISComm_Agenda_20200114.pdf">Download Agenda</a>` : '';
+      var communityMeetingAgenda = e.community_meeting_agenda_url ? ` <a class="btn btn-tnris btn-md" href=${e.community_meeting_agenda_url}>Download Agenda</a>` : '';
```

Comment:
missing quotes, i think it might break if they aren't there
href="${e.community_meeting_agenda_url}"

Reply:
gotchya i'll fix it